From 0cba1642ae7651672e10514ae83a446ba7c3fedc Mon Sep 17 00:00:00 2001
From: Ed Wildgoose <lists@wildgooses.com>
Date: Fri, 6 Apr 2012 17:59:34 +0300
Subject: [PATCH 70/75] Save bgscan parameter into configuration file

Signed-off-by: Ed Wildgoose <lists@wildgooses.com>
intended-for: hostap-1
(cherry picked from commit af548f5320aac429043b54cc737c7895a05a1691)
---
 wpa_supplicant/config_file.c |    1 +
 1 file changed, 1 insertion(+)

--- a/wpa_supplicant/config_file.c
+++ b/wpa_supplicant/config_file.c
@@ -527,6 +527,7 @@ static void wpa_config_write_network(FIL
 	write_pairwise(f, ssid);
 	write_group(f, ssid);
 	write_auth_alg(f, ssid);
+	STR(bgscan);
 #ifdef IEEE8021X_EAPOL
 	write_eap(f, ssid);
 	STR(identity);
